Readme.md

```diff
@@ -12,61 +14,66 @@
    1. yarn start
 
 ## Dev Notes
 
 - `plop <filename>` scaffolds component files inside the `src/components` directory
-- `src/components/generic` are be for reusable components that may be useful for other projects. They should be developed to be completely unaware of their context. Make them 'dumb' (employ inversion of control). If a reusable component is Mermaid-specific, it can go elsewhere.
+- `src/components/generic` are for reusable components that may be useful for other projects. They should be developed to be completely unaware of their context. If a reusable component is Mermaid-specific, it can go elsewhere.
 - `src/components/pages` are for pages or page-like components
-- Styles will be done with Styled Components for easy scoping, speed, and maintainability. Make sure to import using the macro for easier debugging (it results in more human-friendly classnames) `import styled from 'styled-components/macro'`
-- Focus on integration tests, and testing complex pieces of code.
-- Although there is no comprehensive list, tech debt tickets are tracked in Trello with a label, or the title prefix 'Tech debt:'. Most are in the 'Someday' column. Its worth browsing them for context.
-- The fishbelt form page is a gong show. We havent had a refactor cycle to make it more maintaible in between the many various features and bandaids being added to it. Dont copy its approach. The main issue with it is that Formik didnt age will with the requirements as they revealed themselves. The observations table part of the form, thankfully, doesnt use Formik, however using two different approaches for form state cascaded into much more duplication and complication in the code for that page. For some reason the form also loses its dirty/unsaved values when the network status changes, which resulted in a hack that involves saving unsaved values in local storage, which in hindsight was a bad choice (I probably thought it was unavoidable due to props changing, but there are indications that it may be caused by internal state changing). Yuck. Here be dragons.
+- Styles use Styled Components for easy scoping, speed, and maintainability. Make sure to import using the macro for easier debugging (it results in more human-friendly classnames) `import styled from 'styled-components/macro'`
+- Focus on user-focused integration tests, and testing complex pieces of code. 100% test coverage is not a goal for this project.
+- Although there is no comprehensive list, tech debt tickets are tracked in Trello with a label, or the title prefix 'Tech debt:'. Most are in the 'Someday' column.
+- Collect Record form pages use two different approaches to managing form state. Formik had its limitations, so for the observations tables we opted to store form state with a reducer. This inconsistency has lead to some tech debt and complexity that was determined to be acceptable, but its worth knowing when handling things like dirty form state, that there are two states to consider.
```

Comment:
thank god this is refactored

Reply:
Its not, we just use an abstraction now... or did you refactor it?

Reply:
I meant switching to the same abstraction (Fish belt and BPQ from the old collect record pages)